sycl/CMakeLists.txt

```diff
@@ -129,6 +110,17 @@ add_custom_command(
 install(DIRECTORY "${sycl_inc_dir}/sycl" DESTINATION ${SYCL_INCLUDE_DIR} COMPONENT sycl-headers)
 install(DIRECTORY "${sycl_inc_dir}/CL" DESTINATION ${SYCL_INCLUDE_DIR}/sycl COMPONENT sycl-headers)
 
+# Copy OpenCL Headers into sycl headers build directory
+# Compiler does automatic lookup bin/../include based on clang binary location,
+# e.g. when run LIT tests
+file(COPY ${OpenCL_INCLUDE_DIR}/CL
```

Comment:
It is unclear how this change connected to the PR description.

Reply:
Fair point, I've restored the original placement of the code with 11e4c15e2e989dced603a2831670bcc293609d5. I made the change instinctively due to the mutual usage of `SYCL_INCLUDE_BUILD_DIR/sycl`, however there is no dependence on the `make_custom_target()` call here.